controllers/src/test/scala/cromwell/pipeline/controller/UserWithCredentialsControllerTest.scala

```diff
@@ -35,9 +35,10 @@ class UserControllerTest
 
       "return the sequence of users" taggedAs Controller in {
         val usersByEmailRequest: UserEmail = UserEmail("someDomain@mail.com", Enable.Unsafe)
-        val dummyUser: User = TestUserUtils.getDummyUser()
+        val dummyUser: UserWithCredentials = TestUserUtils.getDummyUser()
```

Comment:
Now once you renamed `User` to `UserWithCredentials` it is clear that we have a lot of places that use full `UserWithCredentials` although they shouldn't do it. <br>For example, here we definitely don't need full `UserWithCredentials` and need only simple `User`. So why bother creating it? I suggest you to tweak `TestUserUtils.getDummyUser()` to return `User` (the one without creds) and create separate `TestUserUtils.getDummyUserWithCredentials()` for some places where we really need a full user.<br>
tip: after you introduce `TestUserUtils.getDummyUserWithCredentials()`, you can re-use in `TestUserUtils.getDummyUser()`

Reply:
@myazinn done